user-org-actor/src/main/java/org/sunbird/OperationValidator.java

```diff
@@ -0,0 +1,16 @@
+package org.sunbird;
+
+/** this is a Util class for actors. */
+public class OperationValidator {
+
+  /**
+   * this method is used to compare the operation names for actor
+   *
+   * @param actorOperation
+   * @param requestOperation
+   * @return boolean
+   */
+  public static boolean matchTwoOperations(String actorOperation, String requestOperation) {
```

Comment:
matchTwoOperations this name looks bit odd , its better rename this to matchOperations(op1,op2)